optuna_dashboard/ts/components/GraphTimeline.tsx

```diff
@@ -65,13 +66,38 @@ const plotTimeline = (trials: Trial[], mode: string) => {
       )
     )
   )
-  const maxDatetime = new Date(
-    Math.max(
-      ...lastTrials.map(
-        (t) => t.datetime_start?.getTime() ?? minDatetime.getTime()
-      )
-    )
+  const maxRunDuration = Math.max(
+    ...trials.map((t) => {
+      return t.datetime_start === undefined || t.datetime_complete === undefined
+        ? -Infinity
+        : t.datetime_complete.getTime() - t.datetime_start.getTime()
+    })
   )
+  const hasRunning =
+    (maxRunDuration === -Infinity &&
+      trials.some((t) => t.state === runningKey)) ||
+    trials.some((t) => {
+      if (t.state !== runningKey) {
+        return false
+      }
+      const start = t.datetime_start?.getTime() ?? new Date().getTime()
+      const now = new Date().getTime()
```

Comment:
[nits] We should define `now` on the top of this function, and use it instead of `Date()` everywhere to have consistent current time.

Reply:
Done